octoprint/tests/test_octoprint.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+
+@pytest.mark.unit
+def test_check():
+    return True
```

Comment:
We still recommend putting at least a real test in here. See @ChristineTChen 's comment on another integration: https://github.com/DataDog/integrations-extras/pull/766#discussion_r541186473.

Here's that PR's completed tests: https://github.com/DataDog/integrations-extras/blob/master/php_apcu/tests/test_php_apcu.py